packages/core/src/resend/audience.ts

```diff
@@ -1,28 +1,19 @@
 /**
- * Minimal helpers for working with Resend Audience contacts without depending on
- * the Resend SDK types. Accepts a "contacts"-like client with a `get` method.
- *
- * These helpers are intentionally tiny and tolerate unknown SDK response shapes
- * by using runtime guards on `unknown` instead of `any`.
+ * Minimal helpers for working with Resend Audience contacts without importing
```

Comment:
@CharlieHelps there is nothing wrong with importing the resend sdk! Please just use the native types instead of inventing your own here